Content.Server/Cargo/Components/ContentsPriceComponent.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Content.Server.Cargo.Components;
+
+/// <summary>
+/// This is used for appraising the price of the contents of a container.
+/// </summary>
+[RegisterComponent]
+public sealed class ContentsPriceComponent : Component { }
```

Comment:
do `ContentsPrice`, `MaterialPrice` and `StackPrice` need to exist? i'd kind of assume that they would just get done automatically when calculating the prices but I guess having the mostly empty comps to direct events to is maybe cleaner. just feels weird if you have something that should reasonably be calculating price based on contents but isn't because it just doesn't have the component, I can't think of a case where you wouldn't want it to recursively do that

Reply:
I can think of one: Secure briefcases, which obscure their contents. This'd prevent people from cheesing it with the appraisal tool.

Reply:
fair enough

Reply:
but in that case it should probably be a component to disallow that from happening rather than the other way round